ESQL Docs: Mention Discover/Field Statistics in OOM known issue in 8.15.1/2 (#113196)
elastic/elasticsearch: docs/reference/release-notes/8.15.0.asciidoc

** These indices have many conflicting field mappings
** Many of those fields are included in the request
These issues deplete heap memory, increasing the likelihood of OOM errors. (issue: {es-issue}111964[#111964], {es-issue}111358[#111358]).

> In Kibana, such a query can be executed indirectly by navigating to Discover, or when adding a Field Statistics panel while editing a dashboard.

Reviewer: ```suggestion
In Kibana, you might indirectly execute these queries when using Discover, or adding a Field Statistics panel to a dashboard.
```
